hazelcast/src/test/java/com/hazelcast/jet/impl/connector/HazelcastConnectorTest.java

```diff
@@ -106,6 +109,46 @@ public void before() {
         cacheManager.getCache(streamSinkName);
     }
 
+    public void when_readNonExistedMap_then_mapNotCreated() {
+        Pipeline p = Pipeline.create();
+        p.readFrom(Sources.map(sourceName))
+                .writeTo(Sinks.noop());
+
+        instance().getJet().newJob(p).join();
+
+        assertTrue(instance().getDistributedObjects().stream().noneMatch(d -> d.getName().equals(sourceName)));
```

Comment:
We could use better assert here, e.g.

```
assertThat(instance.getDistributedObjects())
                .extracting(DistributedObject::getName)
                .doesNotContain(sourceName);
```

It would give a better error message in case of failure and is easier to read.

Reply:
Also in other places where this is used